configure

```diff
@@ -3510,6 +3516,32 @@ if test -n "$_host"; then
 			append_var LDFLAGS "-static-libgcc"
 			append_var LIBS "-laudio"
 			;;
+		miyoo)
+			_sysroot=`$CXX --print-sysroot`
+			_sdlpath=$_sysroot/usr/bin
+			append_var DEFINES  "-DMIYOO -DREDUCE_MEMORY_USAGE -DDINGUX -DOPENDINGUX -DUNCACHED_PLUGINS"
+			append_var CXXFLAGS "-march=armv5te -mtune=arm926ej-s -ffast-math -fomit-frame-pointer -ffunction-sections -fdata-sections"
+			append_var LDFLAGS  "-march=armv5te -mtune=arm926ej-s -ffast-math -fomit-frame-pointer -ffunction-sections -fdata-sections"
+			append_var LDFLAGS  "-O3 -Wl,--as-needed,--gc-sections"
+			_vkeybd=yes
+			_alsa=no
+			_vorbis=yes
+			_mt32emu=no
+			_timidity=no
+			_fluidsynth=no
+			_theoradec=yes
```

Comment:
Is there a reason for explicitly enabling this?

Reply:
Nope, just copy-pasta. Fixed